### ISSUE TITLE
Calculating width and height of SVG with % is not working

### ISSUE THREAD

**Issue by festim-prebreza** [felixfbecker/dom-to-svg#104]

**Comment by felixfbecker** [comment]
Yeah, this is because `<svg>`s are inlined. There would need to be logic added here to handle various elements and their horizontal and vertical attributes and resolve them relative to the viewPort, similar to how it's already done for percentages in presentation attributes (which are resolved relative to the diagonal of the viewBox).

https://github.com/felixfbecker/dom-to-svg/blob/a9309e431ff0d1ae0080f5a1165b70e99d89eae4/src/svg.ts#L128-L134
